scripts/west_commands/sign.py

```diff
@@ -411,6 +413,11 @@ def sign(self, command, build_dir, build_conf, formats):
         if not target:
             log.die('rimage target not defined')
 
+        # user specified signing schema take precedence of the default one
+        # in boards/xtensa/intel_adsp_cavs*/board.cmake
+        if args.sign_schema:
+            target = args.sign_schema
```

Comment:
If you need subclass-specific arguments, please propose a more generic method for adding them in each subclass, so that people who only use imgtool don't need to know about this

Reply:
Hi, I understand your point to keep tool specific options local to the class,  but I am not sure how to do this, because I am not very familiar with the west framework or Python.  Can you suggest a specific solution?

Reply:
@aiChaoSONG please search for `tool_args` in this `sign.py` file, it seems appropriate.